### PR TITLE
Django models cleanup

### PR DESCRIPTION
**Pull request by magul**

### CODE REVIEW

File: backend/fleet_management/models.py [hunk 1]
```diff
@@ -3,31 +3,31 @@
...
     def __str__(self):
         return self.plates
 
 
-class PassengerModel(models.Model):
-    first_name = models.CharField(max_length=20, blank=False, unique=True)
-    last_name = models.CharField(max_length=20, blank=False, unique=True)
+class Passenger(models.Model):
+    first_name = models.CharField(max_length=60, blank=False, unique=True)
+    last_name = models.CharField(max_length=60, blank=False, unique=True)
```
Comment (juliaanholcer) [inline backend/fleet_management/models.py]: Is `unique=True` a good idea?

Comment (magul) [reply backend/fleet_management/models.py]: Haven’t seen that. Good catch!

File: backend/fleet_management/models.py [hunk 2]
```diff
@@ -3,31 +3,31 @@
...
         return self.last_name
 
 
-class RouteModel(models.Model):
-    driver = models.ForeignKey(DriverUserProfile, on_delete=models.CASCADE)
-    car = models.ForeignKey(CarModel, null=False, on_delete=models.CASCADE)
-    passengers = models.ManyToManyField(PassengerModel)
+class Route(models.Model):
+    driver = models.ForeignKey(Driver, on_delete=models.CASCADE)
+    car = models.ForeignKey(Car, null=False, on_delete=models.CASCADE)
```
Comment (jacol12345) [inline backend/fleet_management/models.py]: More of a product requirement: do we want to delete drivers and cars? I think "disabling" them would be better - we may need the data later.

Comment (magul) [reply backend/fleet_management/models.py]: Most probably we will disable them, but Django since version 2.x requires `on_delete` parameter.

**Comment by magul** [comment]
@juliaanholcer your comment was fixed, @jacol12345 your one was answered to.

**Pull request merged**